flo_detect_intent.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DialogFlow API Detect Intent Python sample with text inputs.
+
+Examples:
+  python detect_intent_texts.py -h
+  python detect_intent_texts.py --project-id PROJECT_ID \
+  --session-id SESSION_ID \
+  "hello" "book a meeting room" "Mountain View"
+  python detect_intent_texts.py --project-id PROJECT_ID \
+  --session-id SESSION_ID \
+  "tomorrow" "10 AM" "2 hours" "10 people" "A" "yes"
+"""
+
+# [START import_libraries]
+import argparse
+import uuid
+
+import dialogflow
+
+# [MY IMPORTS]
+import os
+#os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = './LittleSis-3089dc6d6fb8.json'
+os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = './v2_sa.json'
+
+# [END import_libraries]
+
+
+# [START dialogflow_detect_intent_text]
+def detect_intent_texts(project_id, session_id, texts, language_code):
+    """Returns the result of detect intent with texts as inputs.
+
+    Using the same `session_id` between requests allows continuation
+    of the conversaion."""
+    session_client = dialogflow.SessionsClient()
+
+    session = session_client.session_path(project_id, session_id)
+    print('Session path: {}\n'.format(session))
+
+    for text in texts:
+        text_input = dialogflow.types.TextInput(
+            text=text, language_code=language_code)
+
+        query_input = dialogflow.types.QueryInput(text=text_input)
+
+        response = session_client.detect_intent(
+            session=session, query_input=query_input)
+
+        print('=' * 20)
```

Comment:
are these print statements important for the application? Else you can just remove them before committing or leave them out of the commit